gulpfile.js

```diff
@@ -15,28 +15,46 @@ var phpwatcher = require('./lib/phpwatcher');
 var less = require('./lib/less');
 var lesswatcher = require('./lib/lesswatcher');
 
+var knownOptions = {
+  boolean: 'no.symlinks',
+  default: { 'no.symlinks': false }
+};
```

Comment:
I went with a dot notation since the minimist package appeared to struggle with dashes in argument names (`no-symlinks`).